src/constants/locConstants.ts

```diff
@@ -1914,6 +1928,42 @@ export class MssqlChatAgent {
     public static chatCommandNotAvailable = l10n.t(
         "Chat command not available in this VS Code version",
     );
+
+    // Help command strings
+    public static helpWelcome = l10n.t(
+        "👋 I'm GitHub Copilot for MSSQL extension, your intelligent SQL development assistant in Visual Studio Code. I help you connect, explore, design, and evolve your SQL databases directly from VS Code.",
+    );
+    public static helpWhatICanDo = l10n.t("What I can do for you");
```

Comment:
Syntax nit: I think this should probably have a `:` at the end since you're listing capabilities afterward

Reply:
updated